web/api/v1/api.go

```diff
@@ -909,11 +911,13 @@ type alertingRule struct {
 }
```

Comment:
The `alertingRule` type also needs the new `EvaluationDuration` and `LastEvaluation` fields (and have them set below).